pycbc/distributions/sky_location.py

```diff
@@ -31,5 +35,56 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class Fisher():
+    """A distribution that returns a random (ra, dec) angle drawn from the
+    Fisher distribution. Assume that the concentration parameter (kappa)
+    is large so that we can use a Rayleigh distribution about the north
+    pole and rotate it to be centered at the (ra, dec) coordinate mu.
+    Assume kappa = 1 / (0.66*sigma)**2
+    As in UniformSky, the declination (dec) varies from pi/2 to-pi/2
+    and right ascension (ra) varies from 0 to 2pi. And the angles
+    should be provided in (ra,dec) format in radians (mu_radians=True),
+    rather than factors of pi, or in degrees (mu_radians=False).
+    References:
+      * http://en.wikipedia.org/wiki/Von_Mises-Fisher_distribution
+      * http://arxiv.org/pdf/0902.0737v1 (states the Rayleigh limit)
+    """
+    name = 'fisher'
```

Comment:
@Prasia-Pankunni please change this to `fisher_sky` as well.

Reply:
This is also done